OpenRA.Game/Network/UnitOrders.cs

```diff
@@ -18,6 +18,8 @@ namespace OpenRA.Network
 {
 	static class UnitOrders
 	{
+		private const string ServerChatName = "Battlefield Control";
```

Comment:
I can either remove private or make it public and use it also in class WorldInteractionControllerWidget where "Battlefield Control" literal is used in more places.

Re Coding standard - private variables clearly camelCase, however when you search for const in OpenRA code most (all?) public and private const use CamelCase so I would keep it to be consistent.